@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -728,7 +728,7 @@ export class TileGeometryCreator {
                 if (isFillTechnique(technique)) {
                     object.onBeforeRender = chainCallbacks(
                         object.onBeforeRender,
-                        (_renderer, _scene, _camera, _geometry, _material) => {
+                        (_renderer, _scene, _camera, _geometry, _material, _group) => {
```

Comment:
Why is that even needed?
We don't use this group parameter so, it can be safely removed
Typescript accepts callback that has less arguments than required.

Reply:
At least it does fail for other types of "tsconfigs". We're building harp.gl from source for MapLab and that change throws a `TS2345: Argument of type [...] is not assignable to parameter of type [...]`, I assume, because of something either stricter on our side or permissive on harp.gl's. Seems like a small enough but helpful change to support more TS configurations